index.ts

```diff
@@ -1,7 +1,6 @@
 import { createClient as _createClient, createCluster as _createCluster } from '@node-redis/client';
 import { RedisScripts } from '@node-redis/client/dist/lib/commands';
-import { RedisClientOptions, RedisClientType } from '@node-redis/client/dist/lib/client';
-import { RedisClusterOptions, RedisClusterType } from '@node-redis/client/dist/lib/cluster';
+import { RedisClientOptions, RedisClientType, RedisClusterOptions, RedisClusterType } from '@node-redis/client';
```

Comment:
please merge this import with the first import

Reply:
Merged.